utils/firestore.js

```diff
@@ -2257,16 +2257,16 @@ const processReceiptData = async (collectionIdHolder, collectionIdKeys, dateTime
                 
                 let conceptTube;
                 if (miscTubeIdSet.has(tubeId)) {
-                    const tubeKey = Object.keys(specimenData).find(tubeKey => tubeConceptIds.includes(tubeKey) && specimenData[tubeKey][fieldMapping.objectId] === element);
-                    conceptTube = tubeKeyToNum[tubeKey];
+                    conceptTube = Object.keys(specimenData).find(tubeKey => tubeConceptIds.includes(tubeKey) && specimenData[tubeKey][fieldMapping.objectId] === element);
                 } else {
                     conceptTube = collectionIdConversion[tubeId]; 
                 }
```

Comment:
Is it possible to shorten code like below?
```js
let conceptTube = collectionIdConversion[tubeId];
if (miscTubeIdSet.has(tubeId)) {
  conceptTube = Object.keys(specimenData).find(tubeKey => tubeConceptIds.includes(tubeKey) && specimenData[tubeKey][fieldMapping.objectId] === element);
  }
```

Reply:
Sure, updated.